### PR TITLE
Update about OS language support in the SP docs--PR3

### PR DESCRIPTION
**Pull request by ManikaDhiman**

### CODE REVIEW

File: azure-stack/hci/deploy/deployment-tool-install-os.md [hunk 1]
```diff
@@ -28,14 +28,17 @@ Before you begin, make sure you've done the following:
 
 ## Boot and install the operating system
 
-To install the Azure Stack HCI operating system, follow these steps:
+The Supplemental package supports only the English version of the Azure Stack HCI operating system.
+
```
Comment (alkohli) [inline azure-stack/hci/deploy/deployment-tool-install-os.md:32]: Perhaps reword: Azure Stack HCI Supplemental Package supports only the English version of the Azure Stack HCI operating system.

**Review by alkohli — COMMENTED** [review]
One minor change suggested inline. Also additional articles flagged on the teams channel.

**Comment by Court72** [comment]
@dansisson

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by alkohli** [comment]
#sign-off

**Comment by dansisson** [comment]
#sign-off

File: azure-stack/hci/deploy/deployment-tool-install-os.md (hunk 1 above)
Comment (Jak-MS) [reply azure-stack/hci/deploy/deployment-tool-install-os.md:32]: @ManikaDhiman @alkohli - if you want to make these changes, can you submit them in a new pr? thanks.

**Pull request merged**